constellation_base/signals.py

```diff
@@ -0,0 +1,15 @@
+from django.dispatch import receiver
+from django.contrib.auth.signals import user_logged_in
+
+from .models import LocalGroupACL
+
+
+@receiver(user_logged_in)
+def addSupplementalGroups(sender, user, request, **kwargs):
+    """Apply additional groups on login"""
+    for group in list(LocalGroupACL.objects.all()):
+        supGroup = group.group
+        if user.username in group.members:
```

Comment:
By using the simple check, we achieve what the end goal is without adding unnecessary complexity.  The large field for usernames also satisfies the constraint of being easy to do updates for non-technical people.